CMakeLists.txt

```diff
@@ -21,6 +29,7 @@ endif()
 
 # Set output directories for the library files
 set(CMAKE_LIBRARY_OUTPUT_DIRECTORY ${PROJECT_BINARY_DIR}/lib)
+set(CMAKE_ARCHIVE_OUTPUT_DIRECTORY ${PROJECT_BINARY_DIR}/lib)
```

Comment:
This doesn't seem to be used anywhere, please remove.

Reply:
This is implicitly used by CMAKE, and it is for specifying the output directory for static libs.